rareapi/views/categories.py

```diff
@@ -82,6 +82,24 @@ def update(self, request, pk=None):
         # server is not sending back any data in the response
         return Response({}, status=status.HTTP_204_NO_CONTENT)
        
+    def destroy(self, request, pk=None):
+        """Handle DELETE requests for a single category
+
+        Returns:
+            Response -- 200, 404, or 500 status code
+        """
+        try:
+            category = Categories.objects.get(pk=pk)
+            category.delete()
+
+            return Response({}, status=status.HTTP_204_NO_CONTENT)
+
+        except category.DoesNotExist as ex:
```

Comment:
`category` on this line should be `Categories` because it's referring to the model class